### ISSUE TITLE
a11y CRITICAL: `auro-dropdown` tigger's aria-labelledby element is empty

### ISSUE THREAD

**Issue by sun-mota** [AlaskaAirlines/auro-formkit#613]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [ ] Mobile

### Steps to reproduce

When there is no `label` slot,
Either `label` slot should be required, or change trigger's aria-labelleby target to something else when there is no `label' slot.

```
      Error: Accessibility Violations
      ---
      Rule: aria-input-field-name
      Impact: serious
      ARIA input fields must have an accessible name (https://dequeuniversity.com/rules/axe/4.10/aria-input-field-name?application=axeAPI)
      
      Issue target: #\38 56e1f89-49ce-4637-9814-98531a90f7ef,#trigger
      Context: <div id="trigger" class="trigger" part="trigger" tabindex="0" showborder="" role="combobox" aria-expanded="false" aria-labelledby="triggerLabel" aria-controls="856e1f89-49ce-4637-9814-98531a90f7ef">
      Fix any of the following:
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not e
```

### Reproducible on the Auro doc site

- [ ] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by sun-mota** [comment]
fixed by #664
